_posts/2018-02-27-optimizing-grpc-part-1.md

```diff
@@ -0,0 +1,372 @@
+---
+layout: post
+title: Optimizing gRPC - Part 1
+published: false
+permalink: blog/optimizing-grpc-part-1
+author: Carl Mastrangelo
+company: Google
+company-link: https://www.google.com
+---
+
+# Optimizing gRPC - Part 1
+
+A common question with gRPC is how to make it fast.  The gRPC library offers users access to high
+performance RPCs, but it isn't always clear how to do it.   Because this question is common enough
+I thought I would try and show my thought process when tuning programs.
```

Comment:
s/try and/try to

while _try and_ is colloquially accepted, I think _try to_ is preferred in writing. [arbitrary source](http://www.dictionary.com/e/try-and-or-try-to/)

Reply:
Done.